### ISSUE TITLE
CLI: Activate orchestrator always results in "error: current round is locked"

### ISSUE THREAD

**Issue by kyriediculous** [livepeer/go-livepeer#1648]
**Describe the bug**
After Invoke multi-step "become an orchestrator" follow with
Error activating orchestrator: current round is locked.

The round is not actually locked 

**Expected behavior**
Should only throw error when the roundi s locked 

**Additional context**
the following code https://github.com/livepeer/go-livepeer/blob/9d47b956559b00d07951dd4ae29f4d03969bf08f/server/webserver.go#L234 results in a false positive for the round being locked 
```golang
		ok, err := s.LivepeerNode.Eth.CurrentRoundLocked()
		if err != nil {
			respondWith500(w, err.Error())
			return
		}
		if !ok {
			respondWith500(w, "current round is locked")
			return
		}

```

`if !ok`  should be `if ok` but we should also rename `ok` to `isLocked` for readability

**Comment by dob** [comment]
Sounds like this might be one that necessitates a patch release, right? It's currently blocking anyone from registering as a new O.

**Comment by kyriediculous** [comment]
We could although there's workarounds

- Import your keys in metamask , self-bond using etherscan, then use "Set orch config" option in CLI
- Build from source once this PR is merged (or from this branch)

We could do a quick maintenance release tomorrow though perhaps ? Wondering what @yondonfu thinks.

**Comment by yondonfu** [comment]
Since this bug prevents a user from activating a new O and since outside of building from source there is not a very user-friendly way (considering moving keys from machine as not user-friendly here) to work around this bug, I think it makes sense to publish a new release with a fix for this bug. FWIW we have published new releases to resolve bugs that block users from operating on the network in the past - for example, [0.5.7](https://github.com/livepeer/go-livepeer/releases/tag/v0.5.7) was published to fix a broken Windows build.

**Comment by dob** [comment]
Right. I don't think it needs to rushed out with a drop-everything else mentality, but a core action that's documented and tutorialized, that is blocked because of a bug deserves a patch release as soon as is convenient.

**Comment by chrishobcroft** [comment]
Is this a duplicate of https://github.com/livepeer/go-livepeer/issues/1509 ? @kyriediculous

**Comment by kyriediculous** [comment]
#1509 was fixed in https://github.com/livepeer/go-livepeer/pull/1570 which introduced the codeblock that was the cause of the bug described in this issue.
